From d32f44c3f8a2b3245e73ff2f8389912dceaed13e Mon Sep 17 00:00:00 2001
From: Lucio Correia <luciojhc@linux.vnet.ibm.com>
Date: Thu, 29 Dec 2016 10:12:52 -0200
Subject: [PATCH] Remove redundant kill on ExecStop

Signed-off-by: Lucio Correia <luciojhc@linux.vnet.ibm.com>
---
 contrib/wokd.service.systemd | 1 -
 1 file changed, 1 deletion(-)

diff --git a/contrib/wokd.service.systemd b/contrib/wokd.service.systemd
index d21c845..0628c2d 100644
--- a/contrib/wokd.service.systemd
+++ b/contrib/wokd.service.systemd
@@ -6,7 +6,6 @@ After=nginx.service
 [Service]
 Type=simple
 ExecStart=/usr/bin/wokd
-ExecStop=/bin/kill -TERM $MAINPID
 EnvironmentFile=/etc/wok/wok.conf
 KillMode=process
 
-- 
2.7.4

